lib/generators/solidus_paypal_commerce_platform/install/templates/app/assets/javascripts/spree/frontend/solidus_paypal_commerce_platform.js

```diff
@@ -1,3 +1,10 @@
 //= require spree/frontend/solidus_paypal_commerce_platform/namespace
 //= require spree/frontend/solidus_paypal_commerce_platform/button_actions
 //= require spree/frontend/solidus_paypal_commerce_platform/buttons
+//= require_self
```

Comment:
Nitpicking: for the commit of all these changes, "Fix the specs for SSF" is misleading, as it looks like we're not touching the extension's code. Probably something like the title of the PR, "Setup compatibility with the starter frontend", would be more appropriate.